config/config.php

```diff
@@ -1,11 +1,105 @@
-<?php return [
-      /*
-      |--------------------------------------------------------------------------
-      | Minimum Password Length
-      |--------------------------------------------------------------------------
-      |
-      | The minimum length of characters required for user passwords.
-      |
-      */
+<?php
+
+use RainLab\User\Models\Settings;
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Activation mode
+    |--------------------------------------------------------------------------
+    |
+    | Select how a user account should be activated.
+    |
+    */
+
+    'activateMode' => Settings::ACTIVATE_AUTO,
```

Comment:
Could you also document all the available options for this configuration value and explain them?

Reply:
Resolved e6bf7a2bb5f12cbebffb32bed1ff9b58ce5c429b